octobot_trading/data/order.py

```diff
@@ -153,43 +158,28 @@ def update(self, symbol, order_id="", status=OrderStatus.OPEN,
                 self.executed_time = self.exchange_manager.exchange.get_uniform_timestamp(timestamp)
         return changed
 
-    async def update_order_status(self, last_prices: list):
+    async def initialize_impl(self):
+        asyncio.get_event_loop().call_later(self.CHECK_ORDER_STATUS_AFTER_INIT_DELAY, self.update_order_status)
```

Comment:
Should this be called immediatly in backtesting ?

Reply:
Right ! :+1: